app/views/profile/index.html.slim

```diff
@@ -1,5 +1,4 @@
-- content_for(:nav) do
-  = render 'shared/nav_auth'
+= shared_nav(greeting: @view_model.header_personalization)
```

Comment:
Cool, I will do that in a followup too, it means touching another 8 files. For now I moved the content_for back into the profile index view!